joomla/joomla-cms
# Incorrect position of calendar in Internet Explorer 7-10 (Rework #3079)

This is a rework of the PR by @piotrmocko: #3079

Due to conflicts in the compressed JS file and the number of commits it was easier to create a new, clean PR for this issue.

**How to test**
1. Download, install and activate the test module: https://github.com/joomla/joomla-cms/pull/3079#issuecomment-125932042
2. Click on the calendar button in the frontend in the Internet Explorer (7-10) and you will see that the calendar is not displayed directly above the field but much higher. Do the same with a proper browser and you will see that it works as it should.
3. Apply the patch and check again with IE.
   (For more insight, please check the origin PR.)

All kudos to @piotrmocko!
